includes/Services/TemplatePartsService.php

```diff
@@ -0,0 +1,102 @@
+<?php
```

Comment:
I think we can remove this file as well?

Reply:
I guess this is also being used in GlobalStyles for the DIY flow